database/meta.go

```diff
@@ -134,11 +130,6 @@ func (m *MetaDB) DeleteDB(name string) error {
 		return err
 	}
 
-	_, err = ssn.DB(m.config.S.Bro.MetaDB).C("files").RemoveAll(bson.M{"database": name})
```

Comment:
This chunk was doing the exact same thing as the few lines above it, with the difference that "files" is hard-coded here instead of using the value from the config as above.